Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -384,8 +382,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Checks whether any host-related anomalies were found in the investigation
-        (the host is risky or malware alerts occurred on the host in the past 1 day).
+      description: Checks whether any host-related anomalies were found in the investigation (the host is risky or malware alerts occurred on the host in the past 1 day).
```

Comment:
Checks whether any host-related anomalies were found in the investigation (the host is risky or malware alerts occurred on the host in the past day).